From 66df3c2bc482bd2f444a3cf1dd56d63ae6030e32 Mon Sep 17 00:00:00 2001
From: Guillaume Tucker <guillaume.tucker@collabora.com>
Date: Fri, 14 May 2021 10:37:28 +0100
Subject: [PATCH] CHROMEOS Set defaults for chromeos.kernelci.org

Adjust the default config settings for chromeos.kernelci.org until
this gets a proper solution with Ansible parameters.

Signed-off-by: Guillaume Tucker <guillaume.tucker@collabora.com>
---
 app/dashboard/__init__.py         | 2 +-
 app/dashboard/default_settings.py | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/app/dashboard/__init__.py b/app/dashboard/__init__.py
index 671a8aaa99c5..1e9064803698 100644
--- a/app/dashboard/__init__.py
+++ b/app/dashboard/__init__.py
@@ -51,7 +51,7 @@ __versionfull__ = __version__
 
 CSRF_TOKEN_H = "X-Csrftoken"
 
-DEFAULT_CONFIG_FILE = "/etc/kernelci/kernelci-frontend.cfg"
+DEFAULT_CONFIG_FILE = "/etc/kernelci/chromeos.kernelci.org/kernelci-frontend.cfg"
 # Name of the environment variable that will be lookep up for app
 # configuration parameters.
 APP_ENVVAR = "FLASK_SETTINGS"
diff --git a/app/dashboard/default_settings.py b/app/dashboard/default_settings.py
index 336ceca6b380..56396ca35aea 100644
--- a/app/dashboard/default_settings.py
+++ b/app/dashboard/default_settings.py
@@ -22,8 +22,8 @@
 # along with this library; if not, write to the Free Software Foundation, Inc.,
 # 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
 
-LOGGER_NAME = "kernelci-frontend"
-SESSION_COOKIE_NAME = "kernelci.org"
+LOGGER_NAME = "chromeos-kernelci-frontend"
+SESSION_COOKIE_NAME = "chromeos.kernelci.org"
 
 # Following keys should be defined in an external file and passed as an
 # environment variable called FLASK_SETTINGS, or in /etc/kernelci in a file
-- 
2.20.1

